File: app/src/main/java/com/penseapp/acaocontabilidade/chat/adapters/ChatsAdapterView.java
package com.penseapp.acaocontabilidade.chat.adapters;

import com.penseapp.acaocontabilidade.chat.model.Chat;

/**
 * Created by unity on 03/02/17.
 */

public interface ChatsAdapterView {
    void subscribeForChatsUpdates();
    void unsubscribeForChatListUpdates();
    void onChatAdded(Chat chat);
    void onChatChanged(Chat chat);
    void onChatRemoved(String chatId);
}

File: app/src/main/java/com/penseapp/acaocontabilidade/chat/chats/view/ChatsAdapterView.kt
package com.penseapp.acaocontabilidade.chat.chats.view

import com.penseapp.acaocontabilidade.chat.chats.model.Chat

/**
 * Created by unity on 03/02/17.
 */

interface ChatsAdapterView {

    fun subscribeForChatsUpdates()

    fun unsubscribeForChatListUpdates()

    fun onChatAdded(chat: Chat)

    fun onChatChanged(chat: Chat)

    fun onChatRemoved(chatId: String)
}
